docs/WritingTests.asciidoc

```diff
@@ -871,6 +871,16 @@ from a virtio serial port. A distribution specific call back will be made
 which allows os-autoinst to log into a serial terminal session running on the
 SUT. Once +select_console+ returns you should be logged into a TTY as root.
 
+NOTE: for https://github.com/os-autoinst/os-autoinst-distri-opensuse[os-autoinst-distri-opensuse]
+tests instead of using `select_console('root-virtio-terminal')` directlty is
```

Comment:
s/directlty/directly/

Reply:
@Martchus thx, fixed, with other typos. To be honest, not sure about grammar with definite article vs. indefinite articles (the vs. a)